Apps/WebClient/src/ClientApp/src/components/timeline/immunization.vue

```diff
@@ -19,6 +18,11 @@ export default class ImmunizationTimelineComponent extends Vue {
     private get entryIcon(): IconDefinition {
         return faSyringe;
     }
+
+    private getDisplayFormat(value: string): string {
+        if (value === "") return "N/A";
```

Comment:
This belongs in the ImmunizationTimelineEntry sine that sets the data to be viewed

Reply:
code has been moved to ImmunizationTimelineEntry.